### PR TITLE
Upgrade UUID module.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

Upgrade the UUID module.

**- Test plan**

~~@erquhart @Benaiah I really have no idea how to test this?~~

Tested manually -- the affected components (editorial workflow and RelationControl) still work normally.

**- Description for the changelog**

Upgrade the UUID module.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/actions/editorialWorkflow.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import uuid from 'uuid';
+import uuidv4 from 'uuid/v4';
```
Comment (erquhart) [inline src/actions/editorialWorkflow.js]: Nitpick: let's still refer to it as `uuid`, the implementation detail isn't relevant across the source.

**Comment by tech4him1** [comment]
@erquhart Changed. I'm still not sue how to test this at all, should I just run an editorial workflow site and make sure that it still works?

File: src/actions/editorialWorkflow.js (hunk 1 above)
Comment (tech4him1) [reply src/actions/editorialWorkflow.js]: Changed.

**Comment by Benaiah** [comment]
@erquhart it looks like your requested change has been made - could you rereview?

**Pull request merged**